### ISSUE TITLE
Plugin not reading properly the import/ignore field?

### ISSUE THREAD

**Issue by javierguzman** [microsoft/vscode-eslint#1341]
Hello all,

I have started getting a red highlight when importing images using aliases, with the error "import/no-extraneous-dependencies".  My webpack build is ok and it does not complain when running eslint with webpack eslint plugin. 

In order to get rid of this error, I added `"import/ignore": ["*.png", "*.jpg"]`  and **that made the error to go away**. However, I started getting webpack errors when running my build, specifically:

```
ERROR in Invalid regular expression: *.png: Nothing to repeat
Occurred while linting blabla\index.js:2
```

So I changed my import/ignore to  `["/*.png/", "/*.jpg/"] `and later to `["\\.png$", "\\.jpg$"]` which made the webpack works ok but again VSCode highlights as red the import.

This makes me think VSCode is not reading properly the import/ignore as eslint expects a regular expression but the red goes only away if I use a the glob *png.

I have an issue opened as well in the eslint-plugin-import:
https://github.com/import-js/eslint-plugin-import/issues/2230

I have the following repo where I get the same result:
https://github.com/javierguzman/eslint-error

You may need to do npm install --force for the node_modules.

In order to reproduce the error:

1. Open Header.tsx and you will see no error highlights
2. Run "npm run build:production"
3. You get the invalid regular expression
4. Change the expression of import/ignore
5. Run "npm run build:production" and it is ok
6. However if you go to Header.tsx you will see the import marked as red due to extraneous dependencies

Any help would be really appreciated. 

Thank you in advance and regards.

**Comment by dbaeumer** [comment]
What happens if you execute eslint on the command line and validate the file there  (e.g. `./node_modules/.bin/eslint ....`)

**Comment by dbaeumer** [comment]
The eslint extension honors everything the command line eslint honors. But this that are specific to webpack are not honored.

**Comment by javierguzman** [comment]
I have run npx eslint ./src/features/Header/Header.tsx -c ./config/.eslintrc.json and indeed I get the same error. I do not understand why with webpack is ok and with cli is not. I am completely lost.

**Comment by dbaeumer** [comment]
Two options I know of:

- webpack uses a different version of eslint, plugins, ...
- there is a eslint config somewhere inside of the webpack files,

I will close the issue since the design of the extension is to respect the same config / setting as eslint in the terminal does.
